src/v3/preact.config.js

```diff
@@ -154,6 +169,34 @@ export default {
       'index.js',
     );
 
+    // config.resolve.alias['preact'] = resolve(
+    //   __dirname,
+    //   'node_modules',
+    //   'preact',
+    //   'dist',
+    //   'preact.min.js',
+    // );
+
+    // config.resolve.alias['preact/compat'] = resolve(
+    //   __dirname,
+    //   'node_modules',
+    //   'preact',
+    //   'compat',
+    //   'dist',
+    //   'compat.js',
+    // );
+
+    config.resolve.alias['nls'] = rootResolve('packages', '@okta/i18n/src/json');
```

Comment:
sort of feeling the boundary of v3 bundle and test app webpack config is not very clear. maybe we should separate them into two configs? 
cc: @lesterchoi-okta